bin/compile

```diff
@@ -28,8 +28,8 @@ export_env_dir() {
 
 export_env_dir
 echo "-----> gem install bundler"
-gem install bundler
+gem install bundler --no-ri --no-rdoc
 echo "-----> bundle install"
-bundle install
+bundle install --without development:test --path vendor/bundle --binstubs vendor/bundle/bin -j4 --deployment
 echo "-----> bundle exec rails webpack:compile"
-bundle exec rails webpack:compile --trace
+RAILS_ENV=production bundle exec rails webpack:compile --trace
```

Comment:
@acarpe – im not sure about some of these improvements. Can you explain them individually? 

- `--no-ri`, `--no-rdoc` seem to be options for `gem` and bundler has no such options, but doing what they mean for gem by default
- `--deployment` seems to imply `--path vendor/bundle`
- Do we care about the binstub directory?
- Setting `RAILS_ENV` to `production` would prevent setting it to something else, like a staging env hosted on Heroku?

`--j4`, `--deployment` and `--without development:test` look good to me!